flutter/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM "gitpod/workspace-full-vnc"
+
+ENV FLUTTER_HOME=/home/gitpod/flutter \
+    PATH=/usr/lib/dart/bin:$FLUTTER_HOME/bin:$PATH
+
+RUN sudo sh -c 'curl https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -' && \
+    sudo sh -c 'curl https://storage.googleapis.com/download.dartlang.org/linux/debian/dart_stable.list > /etc/apt/sources.list.d/dart_stable.list' && \
+    sudo apt-get -y update && \
+    sudo apt-get -y install  build-essential dart libkrb5-dev gcc make && \
+    sudo apt-get clean && \
+    sudo apt-get -y autoremove && \
+    sudo apt-get -y clean && \
+    sudo rm -rf /var/lib/apt/lists/*
```

Comment:
Nit: You might want to add `USER root` above this command, and `USER gitpod` after it. This would allow you to remove the `sudo sh -c 'curl'` trick and all other uses of `sudo` here.